src/python_framework_v02/troute/nhd_io.py

```diff
@@ -666,7 +666,9 @@ def get_usgs_df_from_csv(usgs_csv, routelink_subset_file, index_col="link"):
 
 def get_usgs_from_time_slices_folder(
     routelink_subset_file,
+    run_parameters,
```

Comment:
At this level, let's pass explicitly what we need as scalar values with no dict parameters. That makes this function more useful from other contexts and makes it super clear what we need out of those dictionaries. Same comment as below in the calling function.